src/www/build.action.mjs

```diff
@@ -34,7 +34,16 @@ export async function main(...parameters) {
     new Promise((resolve, reject) => {
       webpack(webpackConfig, (error, stats) => {
         if (error || stats.hasErrors()) {
-          reject(error || 'Unknown Webpack error.');
+          reject(
+            error ||
```

Comment:
I am thinking whether we should use [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to be more specific to `null` values rather than `zero` values.

Reply:
In the case where error is an empty string and there are valid errors on stats, we'd get an empty message